src/Symfony/Bundle/FrameworkBundle/Command/AssetsPipelinePublishCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Asset\Pipeline\AssetPipeline;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Publishes the assets in the pipeline to the final output directory.
+ *
+ * This command is intended to be used during deployment.
+ *
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ *
+ * @final
+ */
+#[AsCommand(name: 'assets:pipeline:publish', description: 'Publishes all pipeline assets to the final public output directory.')]
+class AssetsPipelinePublishCommand extends Command
```

Comment:
what about 
```suggestion
class AssetsPipelineCompileCommand extends Command
```

to me, "publish" sound like "deploy" (I'm not native English speaker tho)
here it is just compiling/dumping the content

Reply:
It *is* an action you take during deploy though? Rails does call this command “precompile”, however…